### ISSUE TITLE
Can the bot send attachments?

### ISSUE THREAD

**Issue by Chewbacca96** [jclg/php-slack-bot#20]
I found article in slack documentation (https://api.slack.com/docs/message-attachments) about attachments, and i have to ask, can your bot do this?

**Comment by jclg** [comment]
No, he can't. This bot only uses the RTM api at the moment https://api.slack.com/rtm

> The RTM API only supports posting simple messages formatted using our default message formatting mode. It does not support attachments or other message formatting modes. To post a more complex message as a user clients can call the chat.postMessage Web API method with as_user set to true.

The bot can be extended to use the Web API in order to send attachments. I think it's a must-have feature and I will try to work on it if I find enough time.

**Comment by robi-bobi** [comment]
Any news on this feature?

**Comment by jclg** [comment]
The issue is still open but unfortunately I don't have time to work on this at the moment.
